op-monitorism/liveness_expiration/monitor.go

```diff
@@ -189,22 +191,24 @@ func (m *Monitor) Run(ctx context.Context) {
 		deadline_date := time.Unix(int64(deadline), 0)
 		formattedDate := deadline_date.Format("Monday, January 2, 2006")
 		// 4. Ensure that the invariant is not broken -> (block.timestamp + BUFFER > lastLive(owner) + livenessInterval) == true
-		if (deadline - now) < 0 {
```

Comment:
This was a potential integer overflow as each variable was uint, `deadline - now` could never be less than 0. If now is greater than deadline, then the value will be close to `18446744073709551615`.
The correct way to do this operation is to use `bits.Sub64` and check the borrow value, if it is not 0 then something went wrong.

Reply:
Thanks greatly appreciated there!